README.md

```diff
@@ -1,9 +1,11 @@
-# EOS COSTA RICA DEVELOPER GUIDES
+# EOS Costa Rica Developer Guides
 ![](https://img.shields.io/github/license/eoscostarica/guias.eoscostarica.io) ![](https://img.shields.io/badge/code%20style-standard-brightgreen.svg) ![](https://img.shields.io/badge/%E2%9C%93-collaborative_etiquette-brightgreen.svg) ![](https://img.shields.io/twitter/follow/eoscostarica.svg?style=social&logo=twitter) ![](https://img.shields.io/github/forks/eoscostarica/guias.eoscostarica.io?style=social)
 
+
+
 EOS Costa Rica Developers Documentation Site.
 
-![Guide front page](static/img/guide-page.png)
+![Guide front page](https://user-images.githubusercontent.com/55892352/179574354-73a0d8de-301f-448b-a2f8-e6bf54f6eb6b.png)
```

Comment:
Lets save this image inside `static/img/`

Reply:
fixed, sorry for the delay, didn't see the comment